### PR TITLE
meme: Enable parallel, add doi,tag,desc

### PR DESCRIPTION
**Pull request by saketkc**

### CODE REVIEW

**Comment by saketkc** [comment]
@iml Any clue, what's going wrong here?

**Comment by iml** [comment]
I don't know about the specific error but Open MPI and OpenMP are two different things. The former is supported through the `:mpi` dependency, with `open-mpi` being the default formula.

**Comment by saketkc** [comment]
@mikemcquaid There seems to be an issue with the `bot` specific to `homebrew-science` PRs.

**Comment by mikemcquaid** [comment]
Just trigger a rebuild if you get that error.

**Comment by saketkc** [comment]
Thanks!
Looks like it's been happening more often today.

**Comment by mikemcquaid** [comment]
We have no control over it, unfortunately. If you can work out how to fix it in the upstream Jenkins Git plugin that'd be good. I suspect it may be related to force-pushing.

**Comment by saketkc** [comment]
Looks like a known issue, with a possible fix?
https://github.com/janinko/ghprb/issues/148#issuecomment-110622022

**Comment by mikemcquaid** [comment]
@saketkc Yeh, I'm not sure if that fix is user-configurable. If you know what settings I should change I'm all ears.

**Comment by saketkc** [comment]
Not really. Not even a Jenkins apprentice.

File: meme.rb [hunk 1]
```diff
@@ -16,10 +20,15 @@ class Meme < Formula
     with other packages.
   EOF
 
+  option "without-openmpi", "Disable parallel support"
+  depends_on "open-mpi" => [:cc] if build.with? "openmpi"
```
Comment (iml) [inline meme.rb]: The `:cc` doesn't apply here. Remove the option and use `depends_on :mpi => [:recommended, :cc]`, which should set the appropriate environment and make mpidir below unnecesssary. There are other MPI implementations, so the `:mpi` dependency is supposed to handle chosing between them.

**Comment by saketkc** [comment]
@iml Can you trigger a build?
Giving up on force push.

**Comment by tdsmith** [comment]
Triggered a rebuild.

**Pull request closed without merging**